Biblio.bib

```diff
@@ -140,19 +149,23 @@ @Article{aron2012bitcoin
 
 @misc{mastercoin2013willett,
 	url = {{https://github.com/mastercoin-MSC/spec}},
-	author = {J. R. Willett},
+	note = {{\url{https://web.archive.org/web/20170810035927/https://github.com/OmniLayer/spec}}},
+	author = {Willett, J. R.},
 	title = {{MasterCoin Complete Specification}},
 	year = {{2013}},
 }
 
 @misc{colouredcoins2012rosenfeld,
-	url = {{https://bitcoil.co.il/BitcoinX.pdf}},
-	author = {Meni Rosenfeld},
-	title = {{Overview of Colored Coins}},
+	url = {{https://github.com/Colored-Coins/Colored-Coins-Protocol-Specification}},
+	note = {{Archive: \url{https://web.archive.org/web/20180220235952/https://github.com/Colored-Coins/Colored-Coins-Protocol-Specification}. Earlier version: \url{https://docs.google.com/document/d/1AnkP_cVZTCMLIzw4DvsW6M8Q2JC0lIzrTLuoWu2z1BE/}}},
```

Comment:
I don't think these links are necessary.

Reply:
The link to the earlier version is helpful as it shows authors that are not on Github.